### ISSUE TITLE
video players and hosting

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-website#204]
For the [Perspectives Videos](https://www.w3.org/WAI/perspective-videos/customizable/) and [video intro to Accessibility and W3C stds](https://www.w3.org/WAI/videos/standards-and-benefits/)  we currently use AblePlayer and YouTube hosted-videos. We provide a link to the videos on the W3C website for those who cannot access YouTube (e.g., China and some company firewalls).

For the Evaluation Overview page where we will have multiple videos, we cannot use multiple instances of AblePlayer, and will use something else with a link to a version with AblePlayer.

For those other videos on the Evaluation Overview page, we considered HTML5 video to use the browser player and YouTube player.
**What are the pros and cons? _Feel free to edit this comment with your name next to what you add._**

**1. HTML5 video to use the browser player**

pros:
* available to those who can't get YouTube _(shawn, shadi)_
* uses web standards ('eat our own dog food') _(hidde)_
* less JavaScript and other sources to download _(hidde)_
* less chance for third parties to include tracking in our pages _(hidde)_

cons:
* uses W3C bandwidth (might not scale with many views) _(hidde)_
* [minor] don't get YouTube Analytics _(shawn)_

info:
* [browser support](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#Browser_compatibility) 
* [Scott Vinkle on accessibility support in browsers](https://scottvinkle.me/blogs/blog/how-accessible-is-the-html-video-player)

**2. YouTube player**

pros:
* works better with keyboard  _(hidde)_
* [minor] get YouTube Analytics _(shawn)_

cons:
* Some don't have access to it (China, firewalls), so would have to use the alternative link _(shawn, shadi)_

info:
* [an accessibility review of YouTube player (Google Doc)](https://docs.google.com/document/d/1sGl-gLg8153M8iynafgGdVqyWy5n_H0CbKiJbllg2Sg/edit#heading=h.nuvi4gyge8ij)

<hr>
Note that we still plan to put all videos both on YouTube and on the W3C server.

**Comment by hidde** [comment]
Added some pros and cons
